server/sonar-web/src/main/js/helpers/README.md

```diff
@@ -0,0 +1,90 @@
+## Making Requests
+
+Since 6.0 all API requests must provide CSRF protection token. In order to
+help you to do so, we provide a set of useful helpers.
+
+### Getting Started
+Let's start with a simple GET request.
+
+```
+window.sonarqube.appStarted.then(function () {
+  // some code here
+  window.SonarRequest
+      .getJSON(window.baseUrl + '/api/issues/search')
+      .then(function (response) {
+        // here 'response' contains the object representing the JSON output
+      });
+});
+```
+
+`window.sonarqube.appStarted` returns a `Promise` telling you when the
+application is stated and all the runtime libraries and hooks are installed.
+
+`window.SonarRequest` contains all the helper methods to do an API requests.
+
+`window.SonarRequest.getJSON` is a simplest helper to do an API call,
+receive some data and parse it as JSON.
+
+### API Documentation
+
+#### `window.sonarqube.appStarted: Promise`
+A promise telling when the application has started.
+You must put your code in the `resolve` path.
+
+#### `window.SonarRequest.request(url: string): Request`
+Start making an API call. `url` must start from `window.baseUrl`.
+Return a `Request` instance which has the following methods:
+* `setMethod(method: string): Request` sets the http method, can be `GET`, `POST`, etc.
+* `setData(data: object): Request` sets the request parameters`
+* `submit(): Promise` sends the request
+
+#### `window.SonarRequest.getJSON(url: string[, data: object]): Promise`
+Send a GET request, get a response, parse it as JSON.
+
+#### `window.SonarRequest.postJSON(url: string[, data: object]): Promise`
+Send a POST request, get a response, parse it as JSON.
+
+#### `window.SonarRequest.post(url: string[, data: object]): Promise`
+Send a POST request, ignore the response.
```

Comment:
to be exact, it could "ignore the response content", as response status code is checked.